dashboards-observability/public/components/trace_analytics/components/dashboard/__tests__/__snapshots__/dashboard.test.tsx.snap

```diff
@@ -251,7 +262,7 @@ exports[`Dashboard component renders dashboard 1`] = `
                   },
                 ]
               }
-              dateFormat="MMM D, YYYY @ HH:mm:ss.SSS"
+              dateFormat=""
```

Comment:
Just curious, why is the dateFormat being sent as an empty string??

Reply:
i think because trace-analytics is now reading advanced settings from dashboards, but that doesn't exist in unit tests